content/en/blog/2022/frontend-overhaul/index.md

```diff
@@ -0,0 +1,181 @@
+linkTitle: OTel Demo Front-end Overhaul
+date: 2022-10-18
+author: "[Oscar Reyes](https://github.com/xoscar) (Tracetest)"
+canonical_url: https://tracetest.io/blog/frontend-overhaul-opentelemetry-demo
+---
+
+One of the OpenTelemetry Project's many Special Interest Groups (SIG) is the
+[OpenTelemetry Community demo SIG](https://github.com/open-telemetry/opentelemetry-demo).
+The SIG supports a set of instrumented microservices and a front-end web app
+which are used to show how to instrument a distributed system with
+OpenTelemetry.
+
+The main focus of the web app is to demonstrate how to instrument an
+application, no matter what programming language, platform or OS it uses. The
+web app also shows different instrumentation techniques: automatic and manual,
+metrics, and baggage. All while following the standards and conventions
+prescribed in the official OTel documentation. More about the specific
+requirements can be
+[found here](https://github.com/open-telemetry/opentelemetry-demo/tree/main/docs/requirements).
+
+My company was focused on becoming part of and embracing the OpenTelemetry
+community. One of our goals this summer was to get more involved with a core
+OpenTelemetry project where we could provide a meaningful contribution. The OTel
+demo was the best match for our goal, as contributing would not only help the
+community, but also provide a great example to test and showcase our product
+with.
+
+The first thing we did was to get in contact with
+[Carter Socha](https://github.com/cartersocha), the organizer of the OTel Demo
+SIG. Carter was really welcoming and helped us identify where our contributions
+could be the most impactful. We started looking at
+[the issue created by Austin Parker](https://github.com/open-telemetry/opentelemetry-demo/issues/39)
+referencing a complete front-end overhaul that would involve moving the
+application away from Go server-side render (SSR) to an architecture that
+included a browser-side client (client-side render or CSR), as well as improving
+the overall style, theme, and user experience.
+
+A fun aspect of the work was the request to move the store from a "normal" store
+to an astronomy store to match the OpenTelemetry project’s overall branding.
+
+Once we got the green light from the rest of the OTel demo SIG, then we started
+working on the different changes that were part of the front-end architecture
+overhaul.
+
+## OpenTelemetry Demo Application Description and Tech Stack
+
+The demo app is an astronomy store, with basic eCommerce features such as a
+shopping cart, currency selector, payment, and checkout. It also includes the
+ability to display promotions (ads) and related products depending on the user’s
+context.
+
+The demo’s stack includes multiple microservices in different languages,
+covering each of the following OTel-supported languages:
+
+- .NET
+- C++
+- Elixir
+- Go
+- Node.js
+- PHP
+- Python
+- Ruby
+- Rust
```

Comment:
Would it make sense to make these links into the corresponding language SIG's landing page in the website?

Reply:
Hello @chalin do you know where I can find the links to the SIG's landing page?

Reply:
I was referring to the listing of languages under https://opentelemetry.io/docs/instrumentation/

Reply:
Done 😄